src/main/java/edu/ucsb/cs156/gauchoride/controllers/RideController.java

```diff
@@ -69,41 +79,4 @@ public Ride postRides(
             return savedRide;

```

Comment:
you should add a check that the rider is actually currentUser. Otherwise, riders can create rides for other riders which is not what we want. Also, check the driver in the ride object is actually a driver because right now you can pick a driver that is not actually a driver but just a user. After these checks pass, then you save it to db.